src/components/ProjectList/ProjectList.tsx

```diff
@@ -118,7 +129,7 @@ const ProjectList: Component<Props> = ({ state, actions }) => {
         </tr>
       </thead>
       <tbody>
-        {projects && projects.map((project: ProjectModel) => createProjectRowItem(project))}
+        {filteredProjects.map((project: ProjectModel) => createProjectRowItem(project))}
```

Comment:
Please also show an indication of how many projects are visible.
Refer to the Employees Table.

Reply:
fixed